26 >1networks

HadrienRenaud opened the pull request:
Everything is updated, everything seems to be working properly. 

It lacks the script function, but it can wait.

PierreBrowne committed:
Modifs Guillaume

PierreBrowne committed:
Update README.md

PierreBrowne committed:
Connexion ReadNetwork - Graphics

PierreBrowne committed:
Corrections

HadrienRenaud committed:
Adding statistic button.

HadrienRenaudLebret commented:
Don't delete it, I might be adding the scritp option in this branch.

HadrienRenaud committed:
Adding Script option

HadrienRenaudLebret commented:
Now @Brownie09 you can merge then delete the branch ! If it is not proposed by Github don't do it.

HadrienRenaud committed:
Adding small options.

HadrienRenaud committed:
Adding : mu, alpha, in options. config file working. Everything seems to be working properly.

HadrienRenaud committed:
Display settings.

PierreBrowne committed:
Réorganisation du code, mise en cohérence

- Amélioration des boutons graphiques
- Déplacement de certaines fonctions entre les fichiers
- Mise sous commentaire de toutes les fonctions et prototypes de NetworkArray
- Ajout d'un fichier "tools" qui rassemble des fonctions diverses

HadrienRenaud committed:
Small fixes in script

HadrienRenaud committed:
Merge fusion ok.

PierreBrowne committed:
Add files via upload

HadrienRenaud committed:
Merge fusion for @Brownie09 's commit

PierreBrowne committed:
Add files via upload
